dbs.tf

```diff
@@ -19,3 +41,309 @@ resource "helm_release" "misarch_catalog_db" {
+  repository = "oci://registry-1.docker.io/bitnamicharts"
+  chart      = "postgresql"
+  namespace  = local.namespace
+
+  values = [
+    <<-EOF
+    fullnameOverride: "${local.order_db_service_name}"
+    image:
+      tag: "${var.POSTGRES_VERSION}"
+    auth:
+      enablePostgresUser: false
+      username: "${var.MISARCH_DB_USER}"
+      database: "${var.MISARCH_DB_DATABASE}"
+      password: "${random_password.misarch_order_db_password.result}"
+    metrics:
+      enabled: true
+    EOF
+  ]
+}
+
+resource "helm_release" "misarch_payment_db" {
+  name       = local.payment_db_service_name
+  repository = "oci://registry-1.docker.io/bitnamicharts"
+  chart      = "postgresql"
+  namespace  = local.namespace
+
+  values = [
+    <<-EOF
+    fullnameOverride: "${local.payment_db_service_name}"
+    image:
+      tag: "${var.POSTGRES_VERSION}"
+    auth:
+      enablePostgresUser: false
+      username: "${var.MISARCH_DB_USER}"
+      database: "${var.MISARCH_DB_DATABASE}"
+      password: "${random_password.misarch_payment_db_password.result}"
+    metrics:
+      enabled: true
+    EOF
+  ]
+}
+resource "helm_release" "misarch_review_db" {
+  name       = local.review_db_service_name
+  repository = "oci://registry-1.docker.io/bitnamicharts"
+  chart      = "postgresql"
+  namespace  = local.namespace
+
+  values = [
+    <<-EOF
+    fullnameOverride: "${local.review_db_service_name}"
+    image:
+      tag: "${var.POSTGRES_VERSION}"
+    auth:
+      enablePostgresUser: false
+      username: "${var.MISARCH_DB_USER}"
+      database: "${var.MISARCH_DB_DATABASE}"
+      password: "${random_password.misarch_review_db_password.result}"
+    metrics:
+      enabled: true
+    EOF
+  ]
+}
+
+resource "helm_release" "misarch_return_db" {
+  name       = local.return_db_service_name
+  repository = "oci://registry-1.docker.io/bitnamicharts"
+  chart      = "postgresql"
+  namespace  = local.namespace
+
+  values = [
+    <<-EOF
+    fullnameOverride: "${local.return_db_service_name}"
+    image:
+      tag: "${var.POSTGRES_VERSION}"
+    auth:
+      enablePostgresUser: false
+      username: "${var.MISARCH_DB_USER}"
+      database: "${var.MISARCH_DB_DATABASE}"
+      password: "${random_password.misarch_return_db_password.result}"
+    metrics:
+      enabled: true
+    EOF
+  ]
+}
+
+resource "helm_release" "misarch_shipment_db" {
+  name       = local.shipment_db_service_name
+  repository = "oci://registry-1.docker.io/bitnamicharts"
+  chart      = "postgresql"
+  namespace  = local.namespace
+
+  values = [
+    <<-EOF
+    fullnameOverride: "${local.shipment_db_service_name}"
+    image:
+      tag: "${var.POSTGRES_VERSION}"
+    auth:
+      enablePostgresUser: false
+      username: "${var.MISARCH_DB_USER}"
+      database: "${var.MISARCH_DB_DATABASE}"
+      password: "${random_password.misarch_shipment_db_password.result}"
+    metrics:
+      enabled: true
+    EOF
+  ]
+}
+
+resource "helm_release" "misarch_shoppingcart_db" {
```

Comment:
ohm, shoppingcart (and all other rust services) use mongo

Reply:
Done